src/platform/user/authentication/utilities.js

```diff
@@ -405,6 +436,41 @@ export async function signupOrVerify({
   const eventIsVerified = allowVerification ? '-verified' : '';
   const event = `${policy}-${eventBase}${eventAuthBroker}${eventIsVerified}`;
 
+  const externalRedirect = isExternalRedirect();
+  const { application } = getQueryParams();
+  // isSignup is true for CreateAccountLink, false for VerifyAccountLink
+  if (isSignup) {
+    // register
+    dataDogLog({
+      name: LOG_NAME.REGISTER_ATTEMPT,
+      payload: newPayload({
+        csp: policy,
+        authBroker: useOAuth ? AUTH_BROKER.SIS : AUTH_BROKER.IAM,
+        authLocation: externalRedirect
+          ? AUTH_LOCATION.USIP
+          : AUTH_LOCATION.MODAL,
+        application,
+        level: 'unknown',
+      }),
+      type: STATUS_TYPE.INFO,
+    });
+  } else {
+    // verify
+    dataDogLog({
+      name: LOG_NAME.VERIFY_ATTEMPT,
+      payload: newPayload({
+        csp: policy,
+        authBroker: useOAuth ? AUTH_BROKER.SIS : AUTH_BROKER.IAM,
+        authLocation: externalRedirect
+          ? AUTH_LOCATION.USIP
+          : AUTH_LOCATION.MODAL,
+        application,
+        level: 'unknown',
+      }),
+      type: STATUS_TYPE.INFO,
+    });
+  }
```

Comment:
You could probably just update the `name` property to use a ternary instead of calling the `dataDogLog` twice in the if/else
```
name: isSignup ? LOG_NAME.REGISTER_ATTEMPT : LOG_NAME.VERIFY_ATTEMPT
```